src/MapViewController.swift

```diff
@@ -128,6 +141,49 @@ public class MapViewController: TGMapViewController, LocationManagerDelegate, TG
     }
   }
 
+  public func display(route: OTRRoutingResult) throws {
+    //TODO: We eventually should support N number of routes.
+    if let routeMarker = currentRouteMarker {
+      //We don't throw if the remove fails here because we want to silently replace the route
+      currentRouteMarker = nil
+      markerRemove(routeMarker)
+    }
+    let routeLeg = route.legs[0]
```

Comment:
Ah, good catch. This is something I meant to address when I was moving stuff into the SDK from the sample app.

Valhalla has a concept of "trips", which can have multiple "legs". Each leg is analogous to a single poly-line in Tangram-es. I actually don't know if we support multi-leg routing in Android... @ecgreb / @sarahlensing might have a better idea?